src/main/target/BEEBRAIN_V2F/config.c

```diff
@@ -100,6 +100,7 @@ void targetConfiguration(void)
     batteryConfigMutable()->batteryCapacity = 250;
     batteryConfigMutable()->vbatmincellvoltage = 28;
     batteryConfigMutable()->vbatwarningcellvoltage = 33;
+    batteryConfigMutable()->setBatteryCellCount = 0;
```

Comment:
Default should be fine

Reply:
Wanted to ask about this. Is the config in an FC target just to overwrite default settings?